File: Lab-09-solution/src/main/java/com/petclinic/core/Pet.java
package com.petclinic.core;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;

@Entity
public class Pet {
    @Id @GeneratedValue
    private Long id;
    private String type;
    private String name;

    public Pet() {
    }

    public Pet(Long id, String type, String name) {
        this.id = id;
        this.type = type;
        this.name = name;
    }

    public Long getId() {
        return id;
    }

    public String getType() {
        return type;
    }

    public String getName() {
        return name;
    }
}

File: spring-core-lab-solutions/Lab-kotlin-solution/src/main/kotlin/com/petclinic/core/Pet.kt
package com.petclinic.core

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
data class Pet(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)  val id: Int?,
    val type: String,
    val name: String,
)
